pages/common/gh-alias.md

```diff
@@ -22,3 +22,7 @@
 - Delete a command shortcut:
 
 `gh alias delete {{alias_name}}`
+
+- Set a shell command to search an open pull request using `fzf` and checkout it:
+
+`gh alias set --shell {{switch}} '{{id="$(gh pr list --limit 1000 | fzf | cut -f1)"; [ -n "$id" ] && gh pr checkout "$id"}}'`
```

Comment:
What is "switch" here?

Reply:
the alias name, I didn't use `co` because is pre-defined as `pr checkout` as a default alias. So, I choose `switch` based on `git switch` subcommand. I think about other names like `open`.

Reply:
I see, so it's basically just an alias so that running `gh switch` opens checks out the newest, open PR?

Reply:
no, it's a finder then checkout
![gh-2021-03-14_17 21 46 mkv](https://user-images.githubusercontent.com/1536740/111082966-e938eb00-84e9-11eb-9aea-90e781fa9a6c.gif)

Reply:
Hehe, sweet!! I think I'll run this command, then, to access this cool shortcut 🚀 

I'd reword it though (see my next suggestion.)